editoast/src/views/v2/path/pathfinding.rs

```diff
@@ -53,11 +53,18 @@ crate::schemas! {
 pub enum PathfindingResult {
     Success {
         #[schema(inline)]
+        /// Path description as block ids
         blocks: Vec<Identifier>,
         #[schema(inline)]
+        /// Path description as route ids
         routes: Vec<Identifier>,
+        /// Path description as track ranges
         track_section_ranges: Vec<TrackRange>,
+        /// Length of the path in mm
         length: u64,
+        /// The path offset of each path item given as input of the pathfinding
+        /// The first value is always `0` (begining of the path) and the last one is always equal to the `length`
```

Comment:
I'm not sure I understand what this field is for and the documentation doesn't help me. Especially, with the last sentence, I understand that the vector is filled with numbers from 0 to length and surely, it's not very useful to send a vector of all integers up to a number.

Reply:
We'll try to rephrase that.

------------

FYI: This field returns the position relative to the waypoint path given as input.

### Example 1:

If we have a train schedule with only two waypoints (starting and ending point). And the computed path is 1km long:
Then the response will be `path_items_positions: [0, 1_000_000]`

### Example 2:

If we have a train schedule with four waypoints (`A`, `B`, `C` and `D`). And the computed path is 1km long:
Then a possible response is `path_items_positions: [0, 250_000, 600_000, 1_000_000]`
This means that `B` is 250m from the start of the path and `C` is 600m away.

Reply:
Oh, then length is not the length of a vec, but the length of a path, indeed, should have thought about that. Maybe `Vec::len()` did trigger me in the wrong way.

To help rephrasing it, maybe you could put the unit in the sentence, because I basically understood indexes were in it.